bootstrap/postgres/configs.go

```diff
@@ -472,62 +475,65 @@ func (cr configRepository) ConnectThing(ctx context.Context, channelID, thingID
 func (cr configRepository) DisconnectThing(ctx context.Context, channelID, thingID string) error {
 	q := `UPDATE configs SET state = $1 WHERE EXISTS (
 		SELECT 1 FROM connections WHERE config_id = $2 AND channel_id = $3)`
-	result, err := cr.db.ExecContext(ctx, q, bootstrap.Inactive, thingID, channelID)
+	_, err := cr.db.ExecContext(ctx, q, bootstrap.Inactive, thingID, channelID)
 	if err != nil {
 		return errors.Wrap(errDisconnectThing, err)
 	}
-	if rows, _ := result.RowsAffected(); rows == 0 {
-		return repoerr.ErrNotFound
-	}
 	return nil
 }
 
-func (cr configRepository) retrieveAll(owner string, filter bootstrap.Filter) (string, []interface{}) {
-	template := `WHERE owner = $1 %s`
-	params := []interface{}{owner}
-	// One empty string so that strings Join works if only one filter is applied.
-	queries := []string{""}
-	// Since owner is the first param, start from 2.
-	counter := 2
+func (cr configRepository) retrieveAll(domainID string, thingIDs []string, filter bootstrap.Filter) (string, []interface{}) {
```

Comment:
Do you think , here need for `retrieveAll` as method of `configRepository` struct ? 
And additonally, this function can be renamed to `buildQueryParams` or `buildRetreiveQueryParams`

Reply:
I think it should remain within the struct for consistency and separation of concerns. The name `retrieveAll` implies that the method is responsible for retrieving data from the database, which is misleading as it just builds the query parameters. So yes, a name like `buildRetrieveQueryParams` more accurately reflects its function.

Reply:
I think it should remain within the struct for consistency and separation of concerns. The name `retrieveAll` implies that the method is responsible for retrieving data from the database, which is misleading as it just builds the query parameters. So yes, a name like `buildRetrieveQueryParams` more accurately reflects its function.